### DISCUSSION TITLE
.Net MAUI Content Page Background Image Over Linear Gradient

### DISCUSSION THREAD

**Discussion by sambasile** [dotnet/maui#14463] [Q&A] [answered]
Is it possible to put a background image SVG (BackgroundImageSource) on top of a linear gradient background for a content page? I seem to be able to have one of the other but not both.

**Comment by jamesmontemagno** [comment, marked as answer]
I would probably make a Grid be the root and then put the Image as first item that spans the page. This will for all intents be a layer on top of it.
